lib/assessment.md

```diff
@@ -41,7 +56,12 @@ What selectors could we write to reference each of the following categories of e
 * Any `p` elements whose parent elements are `div`s
 
 <!-- your answer starts here -->
-
+```CSS
+.big {}
+.contentPane {}
+.important .red {}
+.p {}
```

Comment:
No, this looks for a class called "p". What we want are `p` elements whose parent elements are divs. The right selector is `div > p`.